src/docs/integrations/heroku.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: "Heroku Plugin"
+---
+
+## Create a Heroku application
+
+Create an account on [Heroku](http://www.heroku.com) and create an app.
+
+Follow the [instructions](https://devcenter.heroku.com/articles/git) to set up a Heroku app using the command line tool and deploy process.
```

Comment:
small thing. maybe add some clarification about what setting up a Heroku app means. i.e. 'to set up a Heroku app(doesn't have to be a working webapp, a random text file would work)`

Reply:
Oh huh, when I was trying to deploy mine I actually had an issue getting it to work if it _wasn't_ a working webapp. I'm not sure I see the usefulness of deploying a text file, would that be solely for development purposes?